### ISSUE TITLE
Content-Type builder, error on save new element

### ISSUE THREAD

**Issue by kiwipal** [strapi/strapi#10310]
### Describe the bug

After a fresh install, I create a new collection with juste one item (or many, same problem) and when I hit save, I get the reload message eventually replaced by 'an error occured' popup message. Force reloading the page after a moment reveals that everything worked as planned.

Note : Same issue on Postgres and MongoDB.

### Steps to reproduce the behavior

1. Install Strapi 3.6.2 with docker compose
2. Fill the super admin form and log as super admin
3. Click on "Content Type Builder" on the left menu
4. Click "Add another field
5. Add a new text field, give it a name and click on "finish"
6. Click on the green save button
7. The reload server message is displayed
8. "An error occured" message popup is displayed and the page is not updated
9. The error message disappear, but the loader animation continue forever

Note : no other operation required, this can be reproduced right after the install process.

### Expected behavior

Should reload the content without having to reload the page after a while.

### Console log of the docker container

> [2021-05-12T21:42:21.285Z] debug GET /content-type-builder/components (406 ms) 200
> [2021-05-12T21:42:21.424Z] debug GET /content-type-builder/content-types (255 ms) 200
> [2021-05-12T21:42:21.475Z] debug GET /content-type-builder/reserved-names (271 ms) 200
> [2021-05-12T21:42:21.580Z] debug GET /content-manager/content-types (281 ms) 200
> [2021-05-12T21:42:21.602Z] debug GET /admin/0.f112b47f.chunk.js (16 ms) 200
> [2021-05-12T21:42:43.683Z] info File changed: /srv/app/api/essai/models/essai.settings.json
> [2021-05-12T21:42:43.698Z] debug PUT /content-type-builder/content-types/application::essai.essai (212 ms) 201
> [2021-05-12T21:42:43.721Z] info The server is restarting

### Screenshots

<img width="1438" alt="screenshot" src="https://user-images.githubusercontent.com/5513204/118046516-bb511500-b379-11eb-8f1a-a1b930fa243d.png">

As you can see in the console log, some files are never returned (permissions return 0 bytes, etc).

### System

- Docker on MAC M1 ARM
- Node.js version: 12.20.1
- NPM version: 6.0.0
- Strapi version: 3.6.2
- Database: POSTGRES
- Operating system: MACOS BIG Sur

### Docker compose (copy-paste from strapi documentation, with minor changes to add .env files)

```yml
version: '3'
services:
  strapi:
    image: strapi/strapi
    env_file: .env
    environment:
      DATABASE_CLIENT: ${DATABASE_CLIENT}
      DATABASE_NAME: ${DATABASE_NAME}
      DATABASE_HOST: ${DATABASE_HOST}
      DATABASE_PORT: ${DATABASE_PORT}
      DATABASE_USERNAME: ${DATABASE_USER}
      DATABASE_PASSWORD: ${DATABASE_PASSWORD}
    volumes:
      - ./app:/srv/app
    ports:
      - '${STRAPI_PORT}:${STRAPI_PORT}'
    depends_on:
      - postgres

  postgres:
    image: postgres
    env_file: .env
    environment:
      POSTGRES_DB: ${DATABASE_DB}
      POSTGRES_USER: ${DATABASE_USER}
      POSTGRES_PASSWORD: ${DATABASE_PASSWORD}
    volumes:
      - ./data:/var/lib/postgresql/data
    ports:
        - "${DATABASE_PORT}:${DATABASE_PORT}"
```

### Package.json

```json
{
  "name": "app",
  "private": true,
  "version": "0.1.0",
  "description": "A Strapi application",
  "scripts": {
    "develop": "strapi develop",
    "start": "strapi start",
    "build": "strapi build",
    "strapi": "strapi"
  },
  "devDependencies": {},
  "dependencies": {
    "strapi": "3.6.2",
    "strapi-admin": "3.6.2",
    "strapi-utils": "3.6.2",
    "strapi-plugin-content-type-builder": "3.6.2",
    "strapi-plugin-content-manager": "3.6.2",
    "strapi-plugin-users-permissions": "3.6.2",
    "strapi-plugin-email": "3.6.2",
    "strapi-plugin-upload": "3.6.2",
    "strapi-plugin-i18n": "3.6.2",
    "strapi-connector-bookshelf": "3.6.2",
    "knex": "0.21.18",
    "pg": "8.5.1"
  },
  "author": {
    "name": "A Strapi developer"
  },
  "strapi": {
    "uuid": "4dd8e7b3-1872-41ac-b2e2-d86dc12570c8"
  },
  "engines": {
    "node": ">=10.16.0 <=14.x.x",
    "npm": "^6.0.0"
  },
  "license": "MIT"
}
```

**Comment by kiwipal** [comment]
Same issue when deleting a collection (maybe I should open another issue).

**Comment by derrickmehaffy** [comment]
I've confirmed this because I've seen it also but it's extremely hard to have exact reproduction steps.
@pierreburgy I know you've seen it also
@malgamves / @alexandrebodin we had it happen on the most recent livestorm stream with i18n also.

**Comment by alexandrebodin** [comment]
It looks like the timeout of the watcher is too short and errors out too quickly.

**Comment by kiwipal** [comment]
I have tested on Chrome (no extensions, private mode), safari, firefox... with same issues. Does not seem to be related to a specific browser. I suspected it could be linked to an ad blocker for example, but seems not.

**Comment by derrickmehaffy** [comment]
> I have tested on Chrome (no extensions, private mode), safari, firefox... with same issues. Does not seem to be related to a specific browser. I suspected it could be linked to an ad blocker for example, but seems not.

As alex mentioned above yours there is a timeout watching in the admin panel that queries the `HEAD /_health` to see if the server is alive, the timer on that is probably too short

**Comment by gu-stav** [comment]
Hi. I believe this issue is fixed in the current version of strapi (4.6.1) and I was not able to reproduce it. In case anyone here can I will re-open the issue (feel free to ping me).
